std_msgs/Header header

#[deg]
float64 latitude

#[deg]
float64 longitude

#[meter]
float32 ellipsoidal_height

#[0.01 deg]
int16 roll

#[0.01 deg]
int16 pitch

#[0.01 deg]
int16 heading

#[0.01 m]
uint16 position_rms

#[mm/s]
uint16 zupt_rms

#[0.1 deg]
uint8 heading_rms

uint8 flags
